### PR TITLE
Add basic random color palette generator 

### PR DESCRIPTION
**Pull request by michaelfiber**
This does a bit of fiddling with HSV values as a way to make quick and dirty color palettes that can randomly be monotone, complementary or split complementary.  It isn't guaranteed to make a nice palette or even a legible one.  But if you click and you can't read any more, just click again without moving the mouse!

**Note**: I modified the Makefile a bit so that I could just do `RAYLIB_PATH=/home/myprofile/dev/raylib make all` to build. If that breaks anything else I can undo it.

### CODE REVIEW

**Comment by raysan5** [comment]
@michaelfiber Thanks for the addition! I reviewed the files and you are adding the binary, it shouldn't be there.

Also, I follow the same [raylib coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md) for `rGuiStyler`, could you review the formatting?

File: src/gui_main_toolbar.h [hunk 1]
```diff
@@ -189,9 +189,9 @@ void GuiMainToolbar(GuiMainToolbarState *state)
     state->btnLoadFilePressed = GuiButton((Rectangle){ state->anchorFile.x + 12 + 24 + 4, state->anchorFile.y + 8, 24, 24 }, "#5#");
     state->btnSaveFilePressed = GuiButton((Rectangle){ state->anchorFile.x + 12 + 48 + 8, state->anchorFile.y + 8, 24, 24 }, "#6#");
     state->btnExportFilePressed = GuiButton((Rectangle){ state->anchorFile.x + 12 + 72 + 12, state->anchorFile.y + 8, 24, 24 }, "#7#");
-    GuiDisable();
+    //GuiDisable();
```
Comment (raysan5) [inline src/gui_main_toolbar.h:192]: It can be just deleted.

**Comment by michaelfiber** [comment]
I removed the binary, removed the commented out guienable/disable stuff, and went over trying to match the style guide.  Let me know if there's anything else.

**Comment by raysan5** [comment]
@michaelfiber Thank you very much for the addition! :D

**Pull request merged**